components/prize_pool/commons/prize_pool_import.lua

```diff
@@ -204,6 +206,18 @@ function Import._computeBracketPlacementEntries(matchRecords, options)
 			end)
 		end
 	)
+
+	for slotIndex, slotEntries in pairs(slots) do
+		slots[slotIndex] = Array.filter(slotEntries, function(entry)
+			return not (
+				entry.opponent
+				and entry.opponent.type == Opponent.literal -- in case we ever have a team/player with the id
+				and Opponent.toName(entry.opponent):lower() == BYE_OPPONENT_NAME
```

Comment:
BYE's are always literals right?

Reply:
yes, but we technically could at some point get a team/player that actually uses it as their id
hence the additional check for the type

Reply:
I mean we can skip the last line and just ignore literals